src/v/pandaproxy/api/api-doc/schema_registry.json

```diff
@@ -574,6 +574,12 @@
             "in": "query",
             "required": false,
             "type": "string"
+          },
+          {
+            "name": "subjectPrefix",
```

Comment:
Should this have a description?

Reply:
> Should this have a description?

Probably, but none of the query parameters do, I have a task to improve this.